doc/Command-Reference.md

```diff
@@ -2257,6 +2259,56 @@ The list of the WRED profile fields that are configurable is listed in the below
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#ecn)
 
+## Gearbox
+
+This section explains all the Gearbox PHY show commands that are supported in SONiC.
+
+### Gearbox show commands
+This sub-section contains the show commands that are supported for gearbox phy.
+
+**show gearbox interfaces status**
+
+This command displays information about the gearbox phy interface lanes, speeds and status. Data is displayed for both MAC side and line side of the gearbox phy
+
+- Usage:
+  ```
+  show gearbox interfaces status
+  ```
+
+- Example:
+
+```
+home/admin# show gearbox interfaces status
+  PHY Id    Interface    MAC Lanes    MAC Lane Speed    PHY Lanes    PHY Lane Speed    Line Lanes    Line Lane Speed    Oper    Admin
+--------  -----------  -----------  ----------------  -----------  ----------------  ------------  -----------------  ------  -------
+       1    Ethernet0  25,26,27,28               10G      200,201               20G           206                40G      up       up
+       1    Ethernet4  29,30,31,32               10G      202,203               20G           207                40G      up       up
+       1    Ethernet8  33,34,35,36               10G      204,205               20G           208                40G      up       up
+
+  ```
+
+**show gearbox phys status**
+
+This command displays basic information about the gearbox phys configured on the switch. 
+
+- Usage:
+  ```
+  show gearbox phy status
```

Comment:
Here, the subgroup is `phy` (singular), whereas it is plural elsewhere. Our CLI supports abbreviations (prefix-matching), so this command should work, but I prefer to keep things consistent in the documentation.

Reply:
good catch - commit fa9097f520e04dc3a0a5d8ebb3fe620137dce924